### ISSUE TITLE
0.7.1 doesnt match git status output

### ISSUE THREAD

**Issue by JanisVepris** [facebook/PathPicker#244]
Hi. I'm using OS X, iTerm2, oh-my-zsh and tmux.

Version
```
janisvepris@148 ~/w/project> fpp --version
fpp version 0.7.1
```

Git status output
```
janisvepris@148 ~/w/project> git status
On branch develop
Your branch is up-to-date with 'origin/develop'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   www/.idea/dataSources.local.xml
        modified:   www/.idea/dataSources.xml
        modified:   www/.idea/workspace.xml
        modified:   www/.idea/www.iml
        modified:   www/src/Akahot/SettingsBundle/Entity/Brand.php
        modified:   www/src/Akahot/WebBundle/Resources/public/admin/css/app.css
        modified:   www/src/Akahot/WebBundle/Resources/views/Backend/Banner/create.html.twig
        modified:   www/src/Akahot/WebBundle/Resources/views/Backend/Banner/macros.html.twig
        modified:   www/src/Akahot/WebBundle/Resources/views/Backend/Settings/Brand/_form.html.twig

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        www/.idea/dataSources/
        www/.idea/jsLibraryMappings.xml
        www/.idea/symfony2.xml
```

fpp directory:
```
janisvepris@148 ~/w/project> ls -alh ~/.fpp                                                                                                                                                                                          develop!?
total 32
drwxr-xr-x   5 janisvepris  staff   170B Sep 21 15:58 .
drwxr-xr-x+ 44 janisvepris  staff   1.5K Sep 21 16:15 ..
-rw-r--r--   1 janisvepris  staff   140B Sep 21 16:11 .fpp.log
-rw-r--r--   1 janisvepris  staff    33B Sep 21 16:11 .fpp.sh
-rw-r--r--   1 janisvepris  staff   4.1K Sep 21 16:11 .pickle
```

Attempt to use fpp
```
janisvepris@148 ~/w/project> git status | fpp
No lines matched!!
```

**Comment by pcottle** [comment]
Can you try with `fpp -nfc`? its likely because of that top level www match

**Comment by pcottle** [comment]
Try removing the www here:
https://github.com/facebook/PathPicker/blob/master/src/repos.py#L16

**Comment by pcottle** [comment]
resolved in #246 246

**Comment by pcottle** [comment]
oh whoops, this wasnt the home dir one.

but related. removing `www` from that array will fix it
